src/modules/marketplaces/models/MarketplaceReindexState.ts

```diff
@@ -24,13 +24,28 @@ export class MarketplaceReindexState {
     return this.auctions.findIndex((a) => a.marketplaceAuctionId === auctionId);
   }
 
+  getAuctionIndexByIdentifier(identifier: string): number {
+    return this.auctions.findIndex(
+      (a) =>
+        a.identifier === identifier && a.status === AuctionStatusEnum.Running,
+    );
+  }
+
+  getAuctionIndex(auctionId: number, identifier: string): number {
+    if (auctionId) {
+      return this.getAuctionIndexByNonce(auctionId);
```

Comment:
the name here doesn't seem correct, maybe getAuctionIndexByAuctionId?

Reply:
Method removed because it was confusing and marketplace dependent (krogan)